extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -273,18 +276,15 @@ public function getWikiVariables() {
 
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 
-		try {
-			$wikiVariables['navData'] = $this->getNavigationData();
-		} catch ( Exception $e ) {
+		$navData = $this->getNavigationData();
+		if ( empty( $navData ) ) {
 			\Wikia\Logger\WikiaLogger::instance()->error(
-				'Fallback to empty navigation',
-				[
-					'exception' => $e
-				]
+				'Fallback to empty navigation'
 			);
-			$wikiVariables['navData'] = [ ];
+			$navData = [ ];
 		}
 
+		$wikiVariables['navigation'] = $navData;
```

Comment:
the question is if we want to pursue with flattening navData array structure.
If yes, I rename variables